app/models/collaboration.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: collaborations
+#
+#  id                :bigint           not null, primary key
+#  relationship_type :integer
+#  stakeholder_type  :string
+#  subject_type      :string
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+class Collaboration < ApplicationRecord
+#	belongs_to :stakeholder, polymorphic: true
+#	belongs_to :subject, polymorphic: true
+
```

Comment:
Somewhere here you want to define the `enum` too:

```suggestion
  enum relationship_type: { stakeholder: 0, owner: 1 }
```